pkgs/development/tools/rust/ra-multiplex/default.nix

```
@@ -5,24 +5,24 @@
 , rust-analyzer
 }:
 
-rustPlatform.buildRustPackage {
+rustPlatform.buildRustPackage rec {
   pname = "ra-multiplex";
-  version = "0.2.2";
+  version = "0.2.5";
 
   src = fetchFromGitHub {
     owner = "pr2502";
-    repo = "ra-multiplex";
-    rev = "dcb5f83890cb91016b0a1590cc1b732606bb6ec1";
-    hash = "sha256-Hf4Gj9eXEP4gXiqNV4Jq0oiGLX3DtDF9At1feEZ+bUE=";
+    repo = pname;
```

```suggestion
    repo = "ra-multiplex";
```

Using pname is in general considered an anti-pattern, please undo

---

done. would you mind giving a pointer about why it is anti-pattern?

---

It is tied to override interface. Changing the store path pname should not break the download location

---

https://github.com/NixOS/nixpkgs/issues/277994

---

thank you very much!